### ISSUE TITLE
Unzipping issue

### ISSUE THREAD

**Issue by sonirico** [mini2Dx/mini2Dx#33]
Hi,

I have downloaded the zip file from [here](http://mini2dx.org/downloads/mini2Dx-project-generator-linux.zip) , supposed to be for Linux but when attempting to unzip from command line, unzip command throws:

```
Archive:  mini2Dx-project-generator-linux.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of mini2Dx-project-generator-linux.zip or
        mini2Dx-project-generator-linux.zip.zip, and cannot find mini2Dx-project-generator-linux.zip.ZIP, period.
```

I removed and redownloaded the file many times, always with this md5sum:

```
dccc3498a900b19a5aa2d57d1977b7bd  mini2Dx-project-generator-linux.zip
```

Any idea?

**Comment by tomcashman** [comment]
Thanks for reporting the issue! :) Seems there was an error in transferring the Linux release to the server. I've released 1.0.2 of project generator which you can find in the downloads. The checksum for the Linux release is as follows:

```
48fa20b5fcb85c76dc81a1d885692588   mini2Dx-project-generator-linux.zip
```

**Comment by sonirico** [comment]
Thanks for your early response! Game dev time ;)
